assets/css/global.scss

```diff
@@ -2,21 +2,42 @@
 /* Backend Global Stylesheet */
 /*-----------------------------------------------------------------------------------*/
 
+$sensei-notice-icon-height: 30px;
+$sensei-notice-icon-width: 30px;
+
 /* Sensei Notice */
+.wp-core-ui .notice {
+	&.is-dismissible {
+		padding: 10px 16px;
+	}
+}
+
 .sensei-notice {
 	@import '../shared/styles/wp-admin';
 
+	align-items: center;
 	border-left-color: $primary;
-	padding: 10px;
 	display: flex;
-	align-items: center;
 	min-height: 40px;
 
+	&__icon {
+		background: url( '../images/sensei.svg' ) no-repeat;
+		background-size: $sensei-notice-icon-width $sensei-notice-icon-height;
+		height: $sensei-notice-icon-height;
+		margin-right: 5px;
+		min-width: $sensei-notice-icon-width;
```

Comment:
Since it's variables, I think it's not too bad to repeat the values.
But instead of the min-width, we could use a `flex-shrink: 0`, and instead of the background-size with the fixed values, we could also use `background-size: contain`.

So we just set the values in the `width` and `height`.

WDYT?

Reply:
👍🏻 Done in https://github.com/Automattic/sensei/pull/4314/commits/5875f3e738f5380d6da4aa6894b110d55f4d627e.